src/component/molecules/navbar-menu-right/navbar-menu-right.tsx

```diff
@@ -13,13 +13,19 @@ interface NavbarMenuRightPropsModel {
     navbarSearchButtonToggle: boolean;
     activeStep: number;
     burgerToggle: boolean;
+    cartToggle: boolean;
   };
+  cartQuantity: number | undefined;
   updateSearchButtonState: (value: boolean) => void;
-  cartBadgeContent: number | undefined;
-
+  updateCartButtonState: (value: boolean) => void;
 }
 
-const NavbarMenuRight: FC<NavbarMenuRightPropsModel> = ({ state, updateSearchButtonState, cartBadgeContent }) => {
+const NavbarMenuRight: FC<NavbarMenuRightPropsModel> = ({
```

Comment:
NavbarMenuRightProps is good enough no need to add Model

Reply:
resolve